src/terrain/elevation.js

```diff
@@ -40,7 +41,7 @@ export class Elevation {
      * Altitude above sea level in meters at specified point.
      * @param {MercatorCoordinate} point Mercator coordinate of the point.
      * @param {number} defaultIfNotLoaded Value that is returned if the dem tile of the provided point is not loaded
-     * @param exaggerated
+     * @param exaggerated Whether to apply the styling exaggeration to the resulting elevation
```

Comment:
```suggestion
     * @param {boolean} `true` if styling exaggeration should be applied to the resulting elevation.
```
Not sure if this change is valid – can we improve the specificity of this message by adding a type?

Reply:
Yep, good catch